cookbooks/aws-parallelcluster-slurm/recipes/config/login_nodes_daemon_service.rb

```diff
@@ -0,0 +1,54 @@
+require 'yaml'
+config = YAML.safe_load(File.read(node['cluster']['cluster_config_path']))
+GRACETIME_PERIOD = config["LoginNodes"]["Pools"][0]["GRACETIME_PERIOD"]
+
+# Create configuration file
+template "#{node['cluster']['shared_dir']}/login_nodes_daemon_config.json" do
```

Comment:
Every resource (script, config file, ..) that is expected to be shared among head and login nodes should be placed in `node['cluster']['shared_dir_login_nodes']` because , otherwise, you would share dlogin nodes specific resource with compute nodes.

This is not a functional issue, but we improve the security posture of our product by limiting privileges and access.

Reply:
Done!